<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionSettingWindow</class>
 <widget class="QDialog" name="ActionSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>동작 설정</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #EFEFEF;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>340</width>
     <height>380</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border:1px solid black;
border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>338</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;

QTabBar::tab {
  background: rgb(230, 230, 230); 
  border: 1px solid lightgray; 
  padding: 15px;
} </string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::TextElideMode::ElideLeft</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>단축키 설정</string>
     </attribute>
     <widget class="QLabel" name="label_stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>● 중지 단축키</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_run">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox
{
  selection-color: white;
  selection-background-color: blue;
  color:black;
  border:1px solid black;
  border-radius: 5px;
  padding-left: 3px;
}</string>
      </property>
      <property name="placeholderText">
       <string>선택하세요</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_run">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>● 시작 단축키</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox
{
  selection-color: white;
  selection-background-color: blue;
  color:black;
  border:1px solid black;
  border-radius: 5px;
  padding-left: 3px;
}</string>
      </property>
      <property name="placeholderText">
       <string>선택하세요</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_alt1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Alt</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_shift2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ctrl2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Ctrl</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_alt2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Alt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-20</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>동작의 시작/중지 단축키를 설정하세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_shift1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ctrl1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Ctrl</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>배달시간 설정</string>
     </attribute>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>130</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>● 진행중인 주문</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>개 이상일 시</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_stack">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	border: 1px solid black;
	padding-left:3px;
}



QSpinBox::down-button{
	width: 20px;
	margin-right:1px;
}

QSpinBox::up-button{
	width: 20px;
	margin-right:1px;
}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_default">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	border: 1px solid black;
	padding-left:3px;
}



QSpinBox::down-button{
	width: 20px;
	margin-right:1px;
}

QSpinBox::up-button{
	width: 20px;
	margin-right:1px;
}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>● 기본 설정 시간</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>분</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>175</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>분</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_delay">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>175</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	border: 1px solid black;
	padding-left:3px;
}



QSpinBox::down-button{
	width: 20px;
	margin-right:1px;
}

QSpinBox::up-button{
	width: 20px;
	margin-right:1px;
}</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>175</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color:black;</string>
      </property>
      <property name="text">
       <string>설정시간</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
